src/dom/set.js

```diff
@@ -11,9 +11,11 @@ function set (subject, property, value) {
 	else if (property in subject) {
 		subject[property] = value;
 	}
-	else {
+	else if (property) {
 		subject.setAttribute(property, value);
 	}
+
+	return subject;
```

Comment:
Given that what is exported is the overloaded version of `set()`, doesn't it already return `subject`?

Reply:
True! The problem I was trying to fix was in another function. Thank you.